Bio/SeqIO/SeqXmlIO.py

```diff
@@ -155,9 +169,16 @@ def startEntryFieldElement(self, name, qname, attrs):
             return self.startSpeciesElement(attrs)
         if localname == "description":
             return self.startDescriptionElement(attrs)
-        if localname in ("DNAseq", "RNAseq", "AAseq"):
+        if (
+            localname in ("DNAseq", "RNAseq", "AAseq")
+            and float(self.seqXMLversion) >= 0.2
```

Comment:
Instead of checking `self.seqXMLversion` each time, you can set `startEntryFieldElement` to point to the appropriate function at the moment you detect the SeqXML version.